### PR TITLE
[Feature] extend event

### PR DESCRIPTION
**Pull request by quirk0o**
@adamzima @pbzymek @s1mplex @lkudyba @jtkpiotr

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -70,25 +70,27 @@ def destroy
   end
 
   def confirm
-    Event.confirm_or_create(edit_event_params[:conference_room_id], edit_event_params[:event_id])
+    Event.confirm_or_create(params[:conference_room_id], params[:event_id])
```
Comment (pbzymek) [inline app/controllers/events_controller.rb]: should we have some validation for the params here?

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -70,25 +70,27 @@ def destroy
...
     head :ok
   end
 
   def finish
-    google_event_client.finish(edit_event_params[:conference_room_id], edit_event_params[:event_id])
+    google_event_client.finish(params[:conference_room_id], params[:event_id])
```
Comment (pbzymek) [inline app/controllers/events_controller.rb]: params validation ?

File: app/models/google_calendar/event_wrapper/event.rb [hunk 3]
```diff
@@ -21,35 +28,47 @@ def mark_user_event
       end
 
       def valid?
-        return false unless start_time.present? || end_time.present?
-        start_time.date_time.present? && end_time.date_time.present?
+        return false unless starting.present? || ending.present?
```
Comment (adamzima) [inline app/models/google_calendar/event_wrapper/event.rb]: change to &&

File: app/models/google_calendar/event_wrapper/event.rb [hunk 4]
```diff
@@ -21,35 +28,47 @@ def mark_user_event
...
-      def end_time
-        send(:end)
+      def end_time=(ending)
+        self.end.date_time = ending
       end
 
-      def start_time
-        send(:start)
+      def start_time=(starting)
+        self.start.date_time = starting
```
Comment (adamzima) [inline app/models/google_calendar/event_wrapper/event.rb:56]: :+1:

File: app/models/google_calendar/event_wrapper/event.rb (hunk 3 above)
Comment (quirk0o) [reply app/models/google_calendar/event_wrapper/event.rb]: Done

**Review by adamzima — APPROVED** [review]

**Pull request merged**